connectors/README.md

```diff
@@ -38,3 +38,27 @@ is run as a Google Cloud Function.
 The element templates can be found in the `element-templates` directory of each connector.
 
 Example: [aws-lambda/element-templates](aws-lambda/element-templates/aws-lambda-connector.json)
+
+## Development guidelines
+
+### Create a new Connector
+
+To create a new Connector, simply use the [script](../add_new_connector.sh) or generate a new
+project from the [Maven archetype](../connector-archetype-internal).
+
+Execute from the repository root directory:
+```shell
+chmod +x add_new_connector.sh
```

Comment:
I think, this step could be done in git. [Ref](https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---chmod-x).